### PR TITLE
Fix up EFR32 port readme

### PR DESCRIPTION
**Pull request by applecuckoo**
Redo of #7989

### CODE REVIEW

File: ports/silabs/README.md [hunk 1]
```diff
@@ -115,18 +114,18 @@ At the boot stage, two scripts will be run (if not booting in safe mode). First,
...
 With the boards which support USB mass storage, we can drag the files to the board file system. However, because the EFR32 boards don’t support USB mass storage, we need to use a tool like **Ampy** to copy the file to the board. You can use the latest version of **Ampy** and its  command to copy the module directories to the board.
 
-Refer to the guideline below for installing the **Ampy** tool: 
+Refer to the guide below for installing the **Ampy** tool:
 
-https://learn.adafruit.com/micropython-basics-load-files-and-run-code/install-ampy  
+<https://learn.adafruit.com/micropython-basics-load-files-and-run-code/install-ampy>
 
-## Modules supported ##
+## Built-in modules available ##
```
Comment (jepler) [inline ports/silabs/README.md:125]: an up to date listing similar to this automatically generated at build time, on a dedicated page: https://docs.circuitpython.org/en/latest/shared-bindings/support_matrix.html -- would it make more sense to remove this table? It feels like a potential maintenance burden to have it here.

Comment (applecuckoo) [reply ports/silabs/README.md:125]: This was added in the original SiLabs PR, I have no objections for removing it.

Comment (applecuckoo) [reply ports/silabs/README.md]: Done!

**Review by dhalbert — APPROVED** [review]
Thanks!

**Pull request merged**